src/zjs_buffer.c

```diff
@@ -328,9 +322,10 @@ jerry_value_t zjs_buffer_create(uint32_t size, zjs_buffer_t **ret_buf)
     }
     if (size > maxLength) {
 #ifdef DEBUG_BUILD
-        return RANGE_ERROR("size greater than max length");
+        return zjs_standard_error(RangeError, "size greater than max length",
```

Comment:
not sure if the empty string in non-debug is really needed, might as well consolidate it, since the err str is not that long.